javalib/src/main/scala/java/util/concurrent/ConcurrentHashMap.scala

```diff
@@ -203,34 +203,62 @@ object ConcurrentHashMap {
    * hence the existence of this class, although it forwards all its operations
    * to the inner key set.
    */
-  class KeySetView[K, V] private[ConcurrentHashMap] (inner: Set[K])
+  class KeySetView[K, V] private[ConcurrentHashMap] (innerMap: ConcurrentHashMap[K, V], defaultValue: V)
       extends Set[K] with Serializable {
 
-    def contains(o: Any): Boolean = inner.contains(o)
+    def getMappedValue(): V = defaultValue
```

Comment:
getMappedValue is a public member of KeySetView according to https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ConcurrentHashMap.KeySetView.html#getMappedValue--